src/sections/Meshery/Meshery-integrations/Individual-Integrations/relatedIntegration.js

```diff
@@ -49,6 +50,30 @@ const RelatedIntegrationWrapper = styled.section`
   }
 `;
 const RelatedIntegration = ({ category }) => {
+
+  const data = useStaticQuery(graphql`
+  query {
+    allMdx(filter: { fields: { collection: { eq: "integrations" } } }) {
```

Comment:
Add a filter for `published: true` then you will not have to filter for `InProgress` status later on in the code